nspanel_blueprint.yaml

```diff
@@ -131,20 +131,21 @@ blueprint:
               *SYSTEM settings*
 
               *Select the format for date to display*
-            default: '%d.%m'
+            default: '%A, %d.%m'
             selector:
               select:
                 multiple: false
+                custom_value: true
```

Comment:
Wouldn't it cause a problem that the user can edit the built in options as well?
What happens if they erase some character from the end by mistake? Like `Weekday, DD.MM (ex. "Friday, 22.0`

Reply:
> Wouldn't it cause a problem that the user can edit the built in options as well?

If they edit the built in option that will be considered a new entry and the other entries (even the one used to edit) will still available for selection.

> What happens if they erase some character from the end by mistake? Like `Weekday, DD.MM (ex. "Friday, 22.0`

It will try to parse the string as a format. If that fails, nothing will be shown.
The same will happens if you have a separated field for this.

Reply:
First I played with the `custom_value: true` as well, but I was afraid that it would confuse the users that they can edit the text of the built in options.

Reply:
Yeah, I have the same feeling and I'm not sure which option is best. I still thinking about having lots of options on the drop down, as now this will be quite easy.